.devcontainer/devcontainer.json

```diff
@@ -8,14 +8,12 @@
 		"eamodio.gitlens",
 		"oderwat.indent-rainbow",
 		"ms-python.python",
-		"ms-vsliveshare.vsliveshare",
-		"littlefoxteam.vscode-python-test-adapter"
+		"ms-vsliveshare.vsliveshare"
 	],
 	"settings": {
 		"python.pythonPath": "/usr/local/bin/python",
 		"python.analysis.extraPaths": ["/opt/nautobot"],
 		"python.linting.pylintEnabled": true,
-		"python.linting.enabled": true,
-		"pythonTestExplorer.testFramework": "pytest"
```

Comment:
Keep the testing framework if possible?

Reply:
This configuration is for the Plugin, so if we remove the plugin, we should remove the configuration.